include/verilated_vpi.cpp

```diff
@@ -813,6 +814,8 @@ class VerilatedVpiImp final {
     }
     static void dumpCbs() VL_MT_UNSAFE_ONE;
     static VerilatedVpiError* error_info() VL_MT_UNSAFE_ONE;  // getter for vpi error info
+    static void setDirty(bool dirty) { s().m_dirty = dirty; }
+    static bool isDirty() { return s().m_dirty; }
```

Comment:
```suggestion
    static void setDirty(bool dirty) { s().m_dirty = dirty; }
    static bool dirty() { return s().m_dirty; }
```
Accessors don't usually have "is".  Though occasionally break that rule.

I wonder if vpiEvalNeeded would be more descriptive than ditry?

Reply:
> I wonder if vpiEvalNeeded would be more descriptive than ditry?

I can go that way, however should it just be `evalNeeded()` since this is already in the `VerilatedVpiImp` class?  Also, I'd then think we'd want to `s/clearDirty/clearEvalNeeded` and `s/setDirty/setEvalNeeded`.  How's that sound?

Reply:
Agreed.